index.html

```diff
@@ -302,8 +302,11 @@ <h2>Introduction</h2>
 
       <section id="terminology">
         <h3>Terminology</h3>
-
-        <div data-include="https://w3c.github.io/vc-data-integrity/terms.html"></div>
+        <p>
+Terminology used throughout this document is defined in the
+<a data-cite="VC-DATA-INTEGRITY#terminology">Terminology</a> section of the
+[[[VC-DATA-INTEGRITY]]] specification.
```

Comment:
Intentional, not a duplicate. The triple wrapper `[[[ ]]]` displays the title of the remote spec with a direct link to it; the double wrapper `[[ ]]` displays the ReSpec reference name with a link to the citation in the references section of this spec.
```suggestion
[[[VC-DATA-INTEGRITY]]] [[VC-DATA-INTEGRITY]] specification.
```